docs/windows/runtimeclass-class.md

```diff
@@ -17,78 +17,28 @@ ms.author: "mblome"
   
 ## Syntax  
   
-```  
-template <  
-   typename I0,  
-   typename I1 = Details::Nil,  
-   typename I2 = Details::Nil,  
-   typename I3 = Details::Nil,  
-   typename I4 = Details::Nil,  
-   typename I5 = Details::Nil,  
-   typename I6 = Details::Nil,  
-   typename I7 = Details::Nil,  
-   typename I8 = Details::Nil,  
-   typename I9 = Details::Nil  
->  
-class RuntimeClass : public Details::RuntimeClass<typename Details::InterfaceListHelper<I0, I1, I2, I3, I4, I5, I6, I7, I8, I9>::TypeT, RuntimeClassFlags<WinRt>>;  
-  
-template <  
-   unsigned int classFlags,  
-   typename I0,  
-   typename I1,  
-   typename I2,  
-   typename I3,  
-   typename I4,  
-   typename I5,  
-   typename I6,  
-   typename I7,  
-   typename I8  
->  
-class RuntimeClass<RuntimeClassFlags<classFlags>, I0, I1, I2, I3, I4, I5, I6, I7, I8> : public Details::RuntimeClass<typename Details::InterfaceListHelper<I0, I1, I2, I3, I4, I5, I6, I7, I8>::TypeT, RuntimeClassFlags<classFlags> >;  
-```  
+```
+template <typename ...TInterfaces> class RuntimeClass
+template <unsigned int classFlags, typename ...TInterfaces> class RuntimeClass;
+```
   
 #### Parameters  
- `I0`  
- The zeroth interface ID. (Mandatory)  
-  
- `I1`  
- The first interface ID. (Optional)  
-  
- `I2`  
- The second interface ID. (Optional)  
-  
- `I3`  
- The third interface ID. (Optional)  
-  
- `I4`  
- The fourth interface ID. (Optional)  
-  
- `I5`  
- The fifth interface ID. (Optional)  
-  
- `I6`  
- The sixth interface ID. (Optional)  
-  
- `I7`  
- The seventh interface ID. (Optional)  
-  
- `I8`  
- The eigth interface ID. (Optional)  
-  
- `I9`  
- The ninth interface ID. (Optional)  
-  
  `classFlags`  
- A combination of one or more [RuntimeClassType](../windows/runtimeclasstype-enumeration.md) enumeration values.  The   `__WRL_CONFIGURATION_LEGACY__` macro can be defined to change the default value of classFlags for all runtime classes in the project. If defined, RuntimeClass instances are non-agile dy default. When not defined, RuntimeClass instances are agile by default. To avoid ambiguity always specify the RuntimeClassType::FtmBase or RuntimeClassType::InhibitFtmBase.
+ Optional paramater. A combination of one or more [RuntimeClassType](../windows/runtimeclasstype-enumeration.md) enumeration values.  The `__WRL_CONFIGURATION_LEGACY__` macro can be defined to change the default value of classFlags for all runtime classes in the project. If defined, RuntimeClass instances are non-agile dy default. When not defined, RuntimeClass instances are agile by default. To avoid ambiguity always specify the RuntimeClassType::FtmBase or RuntimeClassType::InhibitFtmBase. Note, if InhibitFtmBase and FtmBase are both used the object will be agile.
+ 
+ `...TInterfaces`  
```

Comment:
You can also put an `Implements` here (rather than an interface). Maybe somebody who knows more can fill in the details.

Reply:
I'll leave that for later.